src/renderer/components/version-select.tsx

```diff
@@ -1,5 +1,5 @@
-import { Button, IButtonGroupProps, MenuItem } from '@blueprintjs/core';
```

Comment:
`IButtonGroupProps` is deprecated - noticed when looking through this file.

Reply:
I'm having deja vu. Didn't we aleady remove this? :dizzy:

Reply:
we removed another instance of `IButtonProps` i think :D